sound/soc/sof/ipc4-topology.c

```diff
@@ -174,6 +174,33 @@ static void sof_ipc4_dbg_audio_format(struct device *dev, struct sof_ipc4_pin_fo
 	}
 }
 
+static const struct sof_ipc4_audio_format *
```

Comment:
commit message:

If a process module is connected to additional copier output
pin, the copier output pin format should be set according to
the corresponding input pin format of the process module.

What happens if the process module can support multiple input formats?
I am not clear at all on how we select the 'best' format then.

Reply:
still confused on this.

Reply:
@plbossart in the previous patch, I mentioned that if there are multiple input formats , the first one will be picked. ATM i dont have the logic for any sort of format matching against runtime params for the non-zero pins. We will need to add this as & when we have the need for such a module.

Reply:
ok, do you mind re-adding this explanation for folks like me who can't remember what they reviewed in the previous patch :-)

Reply:
done